### PR TITLE
🫠  Overhaul

### PR DESCRIPTION
**Pull request by djfarly**
wip

### CODE REVIEW

**Comment by djfarly** [comment]
(needs to be squashed before merge)

File: templates/html-css-js-static/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+- Hint 2
+- ...
+
+Switch to the [index.js](./js/index.js) file and make something great happen!
+
+-->
+
+## Notes
+
+- You only have to touch the `js/index.js` file.
```
Comment (aclaes) [inline templates/html-css-js-static/README.md:33]: (This applies to all template readme files)

I think we should be consistent on how we build a reference to another file from the readme. Right now we have:

Task section
- display filename
- linked with relative path
- not formatted as inline code

Notes section
- display filename with relative path
- no link
- formatted as inline code


Currently it's also inconsistent in the existing exercises

File: sessions/css-basics/button-styles/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# button-styles
+
+## Task
+
+## Notes
+
+- You only have to touch the `index.html` file.
```
Comment (aclaes) [inline sessions/css-basics/button-styles/README.md:7]:
```suggestion
- You only have to touch the [`css/styles.css`](./css/styles.css) file.
```

File: sessions/css-basics/button-styles/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# button-styles
+
+## Task
+
```
Comment (aclaes) [inline sessions/css-basics/button-styles/README.md:4]:
```suggestion

- Open the [`css/styles.css`](./css/styles.css) file and follow the instructions given there.

```

File: sessions/css-flexbox/demo-start/README.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# demo-start
+
+## Task
+
+## Notes
+
+- You only have to touch the `index.html` file.
```
Comment (aclaes) [inline sessions/css-flexbox/demo-start/README.md:7]:
```suggestion
- You only have to touch the [`css/styles.css`](./css/styles.css) file.
- You find the end point for this demo in `sessions/css-flexbox/demo-end`.
```

File: sessions/css-flexbox/demo-end/README.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# demo-end
+
+## Task
+
+## Notes
+
+- You only have to touch the `index.html` file.
```
Comment (aclaes) [inline sessions/css-flexbox/demo-end/README.md:7]:
```suggestion
- This is the final version of the demo
- You find the starting point for this demo in `sessions/css-flexbox/demo-start` 
```

File: sessions/css-flexbox/navigation/README.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
+# navigation
+
+## Task
+
+## Notes
+
+- You only have to touch the `index.html` file.
```
Comment (aclaes) [inline sessions/css-flexbox/navigation/README.md:7]:
```suggestion
## Task

- Open the [`styles.css`](./css/styles.css) file and follow the instructions given there.

## Notes

- You only have to touch the [`css/styles.css`](./css/styles.css) file.
```

File: sessions/css-flexbox/profile/README.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
+# profile
+
+## Task
+
+## Notes
+
+- You only have to touch the `index.html` file.
```
Comment (aclaes) [inline sessions/css-flexbox/profile/README.md:7]:
```suggestion
## Task

- Open the [`styles.css`](./css/styles.css) file and follow the instructions given there.

## Notes

- You only have to touch the [`css/styles.css`](./css/styles.css) file.
```

File: templates/html-css-static/README.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+- Hint 2
+- ...
+
+Switch to the [index.html](./index.html) file and make something great happen!
+
+-->
+
+## Notes
+
+- You only have to touch the `index.html` file.
```
Comment (aclaes) [inline templates/html-css-static/README.md:29]: Most exercises based on  this template are concerned with teaching CSS and changes are to made in the `styles.css` file.

I think we should change this in template

**Review by aclaes — CHANGES_REQUESTED** [review]

File: templates/html-css-js-static/README.md (hunk 1 above)
Comment (djfarly) [reply templates/html-css-js-static/README.md:33]: 👍  lets discuss this @ 10

File: sessions/css-grid/fishes-cards/package.json [hunk 9]
```diff
@@ -1,6 +1,9 @@
 {
-  "name": "css-layout-with-grid-fishes",
-  "version": "1.0.0",
+  "name": "css-grid_fishes-cards",
```
[the remark this answers is not in the document]
Comment (djfarly) [reply sessions/css-grid/fishes-cards/package.json:2]: yes please 🐟

**Review by aclaes — CHANGES_REQUESTED** [review]

**Review by aclaes — APPROVED** [review]

**Pull request merged**